src/coloring/greedy_d1_coloring.jl

```diff
@@ -7,7 +7,7 @@ color using greedy approach. The number of colors
 used may be equal or greater than the chromatic
 number χ(G) of the graph.
 """
-function color_graph(G::VSafeGraph,alg::GreedyD1Color)
+function color_graph(G::VSafeGraph, alg::GreedyD1Color)
     V = nv(G)
```

Comment:
prefer small letters for values (`g` for your graph, `v` for your number of vertices)